library/ssl_tls13_client.c

```diff
@@ -49,6 +49,8 @@ static int ssl_tls13_write_supported_versions_ext( mbedtls_ssl_context *ssl,
                                                    size_t *out_len )
 {
     unsigned char *p = buf;
+    unsigned char versions_len = ( ssl->handshake->min_minor_ver <=
```

Comment:
Minor: alternatively, we could initialize to `0` then increment by 2 every time we write a version - this would avoid duplicating the logic.

Reply:
Yes we could but then we do not know the lengths from the start and thus we would have to update them at the end and all together we would not gain much I'd say.